requirements-app.txt

```diff
@@ -17,26 +17,25 @@ click-datetime>=0.2
 docopt>=0.6.2
 
 # 10 May 2022: Newer versions--even 0.30.3--cause import errors.  Versions < 0.31.0 have
-# security vulnerabilities 40416 and 40399, which are ignored in Makefile.
+# security vulnerability 40416, which is ignored in Makefile.
 eventlet==0.30.2
 
-# 9 June 2022: Flask raises ImportError with higher versions.
-fido2 < 1.0.0
+# This is for hardware based authentication schemes.  (i.e. 2FA)
+# https://developers.yubico.com/WebAuthn/
+fido2~=1.1.0
 
-Flask < 2.0
-Flask-Bcrypt==0.7.1
-Flask-Cors==3.0.10
-Flask-JWT-Extended==4.1.0
-flask-marshmallow==0.11.0
-Flask-Migrate==2.5.2
-git+https://github.com/department-of-veterans-affairs/notification-utils.git@1.0.68#egg=notification-utils==1.0.68
-
-# 5 May 2022: Requiring Flask-SQLAlchemy version 2.3.2, 2.5, or 3.0 causes test failures via database
-# errors about foreign key constraint violations.
-git+https://github.com/mitsuhiko/flask-sqlalchemy.git@500e732dd1b975a56ab06a46bd1a20a21e682262#egg=Flask-SQLAlchemy==2.3.2.dev20190108
+Flask~=2.2.3
+Flask-Bcrypt~=1.0.1
+Flask-Cors~=3.0.10
+Flask-JWT-Extended~=4.4.4
+flask-marshmallow~=0.14.0
+Flask-Migrate~=4.0.4
+Flask-SQLAlchemy~=3.0.3
+git+https://github.com/department-of-veterans-affairs/notification-utils.git
 
 # The production image defined by Dockerfile uses Gunicorn.  The development image defined by Dockerfile.local
 # runs the application with the Flask development server by default.
+# TODO - Should we remove this from this file and install it individually in the Dockerfile?
```

Comment:
see above re TODOs and the need for a corresponding ticket and that ticket number added to the comment.

Reply:
I removed this TODO.  References to Gunicorn appear in other parts of the code, and having it as a requirement might also be useful to people who want to run the app locally.